<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QDialog" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget_main" native="true">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>29</y>
     <width>652</width>
     <height>373</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btnOk">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>330</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>28</height>
     </size>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
    <property name="icon">
     <iconset theme="update.png">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>330</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>28</height>
     </size>
    </property>
    <property name="text">
     <string>Cancle</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="labAppTitle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>软件标题</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtAppTitle">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>511</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtIPCType">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>511</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNVRType">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>511</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labNVRType">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>NVR厂商</string>
     </property>
    </widget>
    <widget class="QLabel" name="labIPCType">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>IPC厂商</string>
     </property>
    </widget>
    <widget class="QLabel" name="labUseVideoOffLineCheck">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>离线检测</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboxVideoOffLineCheckInterval">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>220</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labPollType">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>180</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>轮询画面</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboxDelayTime">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>220</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClearIPC">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>260</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>清空IPC信息</string>
     </property>
    </widget>
    <widget class="QLabel" name="labVideoOffLineCheckInterval">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>220</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>检测间隔</string>
     </property>
    </widget>
    <widget class="QLabel" name="labSec">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>220</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>秒</string>
     </property>
    </widget>
    <widget class="QLabel" name="labDelayTime">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>220</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>缓存时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="labSec1">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>140</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>毫秒</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboxRtspType">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>主码流</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>子码流</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnClearAll">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>260</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>恢复出厂值</string>
     </property>
    </widget>
    <widget class="QLabel" name="labRtspType">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>码流类型</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboxPollSleep">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>140</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cboxPollType">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1画面</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4画面</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9画面</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16画面</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cboxPollInterval">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>180</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClearPoll">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>260</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>清空轮询信息</string>
     </property>
    </widget>
    <widget class="QLabel" name="labSec2">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>180</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>秒</string>
     </property>
    </widget>
    <widget class="QLabel" name="labPollInterval">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>180</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>轮询间隔</string>
     </property>
    </widget>
    <widget class="SwitchButton" name="btnAutoPoll">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>87</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClearNVR">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>清空NVR信息</string>
     </property>
    </widget>
    <widget class="QLabel" name="labPollSleep">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>140</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>轮询延时</string>
     </property>
    </widget>
    <widget class="QLabel" name="labAutoPoll">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>自动轮询</string>
     </property>
    </widget>
    <widget class="SwitchButton" name="btnAutoRun">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>87</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="SwitchButton" name="btnUseVideoOffLineCheck">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>87</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labAutoRun">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>开机启动</string>
     </property>
    </widget>
    <widget class="QLabel" name="labMsec">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>220</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>毫秒</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_title" native="true">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>652</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>28</height>
    </size>
   </property>
   <widget class="QLabel" name="lab_Ico">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lab_Title">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>0</y>
      <width>581</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>系统设置</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QPushButton</extends>
   <header location="global">switchbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
